infrastructure/test.action.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -eu
+#
+# Copyright 2024 The Outline Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+readonly TEST_DIR="${BUILD_DIR}/js/infrastructure/"
+rm -rf "${TEST_DIR}"
+
+# Use commonjs modules, jasmine runs in node.
+tsc -p "${ROOT_DIR}/infrastructure" --outDir "${TEST_DIR}" --module commonjs
+jasmine --config="${ROOT_DIR}/jasmine.json"
```

Comment:
I don't think we need a jasmine config. It's probably easiest to just pass what you need in the command